src/OpenSSL/crypto.py

```diff
@@ -1518,6 +1518,20 @@ def set_flags(self, flags):
         """
         _openssl_assert(_lib.X509_STORE_set_flags(self._store, flags) != 0)
 
+    def set_time(self, vfy_time):
+        """Set verification time to this store.
+
+        .. versionadded: 16.3.0
+
+        :param datetime vfy_time: The verification time to set on this store.
+        :return: ``None`` if the verification time was successfully set.
```

Comment:
I don’t think the qualification is necessary.  I *always* returns None.  It just may raise an error.